src/domain/safe-apps/entities/schemas/safe-app.schema.ts

```diff
@@ -0,0 +1,49 @@
+import { JSONSchemaType } from 'ajv';
+import { SafeApp } from '../../../../routes/safe-apps/entities/safe-app.entity';
+import { SafeAppAccessControl } from '../safe-app-access-control.entity';
+import { SafeAppProvider } from '../safe-app-provider.entity';
+
+export const safeAppProviderSchema: JSONSchemaType<SafeAppProvider> = {
+  nullable: true,
+  type: 'object',
+  properties: {
+    url: { type: 'string' },
+    name: { type: 'string' },
+  },
+  required: ['url', 'name'],
+};
+
+export const safeAppAccessControlSchema: JSONSchemaType<SafeAppAccessControl> =
+  {
+    type: 'object',
+    properties: {
+      type: { type: 'string' },
+    },
+    required: ['type'],
+  };
+
+export const safeAppSchema: JSONSchemaType<SafeApp> = {
+  type: 'object',
+  properties: {
+    id: { type: 'number' },
+    url: { type: 'string' },
+    name: { type: 'string' },
+    iconUrl: { type: 'string' },
+    description: { type: 'string' },
+    chainIds: { type: 'array', items: { type: 'string' } },
```

Comment:
The same as above.

Reply:
See [previous comment](https://github.com/5afe/safe-client-gateway-nest/pull/169#discussion_r1007778058).